src/main/java/seedu/address/model/time/TimeMask.java

```diff
@@ -77,13 +76,15 @@ public void mergeIsolatedEvents(IsolatedEventList isolatedEventList, LocalDate s
 
             if (endTime < 0) {
                 endTime = LAST_HOUR_INDEX;
+                end = end.minusDays(1);
+                System.out.println(end);
```

Comment:
I think it would be better to comment out print statement so that the next person wouldn't have to hunt for where the print statement come from